build_manifest.yml

```diff
@@ -135,13 +135,8 @@ yarn-project-base:
     - ^yarn-project/yarn.lock
     - ^yarn-project/.*/package.json$
   dependencies:
-    - l1-contracts
     - bb.js
-    - noir
     - noir-packages
-    - boxes-files
-    - avm-transpiler
```

Comment:
Same comment as above.

Reply:
Transpilation happens as part of noir-projects, not as part of yarn-project.
We moved noir code out of yarn-project and consume it, so i guess this was a stale dependency.